torchtitan/utils.py

```diff
@@ -36,6 +37,24 @@ def _warn_overwrite_env(env, val):
     os.environ[env] = val
 
 
+def set_determinism(job_config: JobConfig) -> None:
+    """
+    Set Python, PyTorch, CUDA seeds and cudnn settings for reproducibility
+    Use 1 to avoid determinism
+    """
+    if job_config.training.seed == 1:
```

Comment:
let's use `None` instead of `1` to disable determinism, as `1` is a valid seed

Reply:
sounds good.  I went with 1 b/c we use 1 as off for TP etc. and b/c None is not valid in yaml...but None is better and we just avoid having it in toml by making it default in config_manager.py.